homeassistant/components/yolink/switch.py

```diff
@@ -32,35 +34,80 @@ class YoLinkSwitchEntityDescription(SwitchEntityDescription):
     """YoLink SwitchEntityDescription."""
 
     exists_fn: Callable[[YoLinkDevice], bool] = lambda _: True
-    value: Callable[[Any], bool | None] = lambda _: None
-    state_key: str = "state"
+    device_type: str | None = None
```

Comment:
Where is the `device_type` description attribute used?

Reply:
`device_type` is useless, I have removed